src/components/FoodDelivery/FoodDeliveryComponent3.jsx

```diff
@@ -1,11 +1,13 @@
-import React from 'react';
+// You can just uncomment this functions and it just returns h1
 
+// export default function FoodDeliveryComponet3() {
+//   return <h1> I am component 3</h1>;
+// }
 
-export default function FoodDeliveryComponet3() {
-
-  return (
-  <React.Fragment>
```

Comment:
I think for this example we probably don't need to give them too many different ways to do things.  The purpose of having 3 components was to show 2 routes with 3 components and having at least one with an embedded component